TheFourFifths/consus-client
# Rfid story tff205

### Summary

Rfid to student association. 

### Checklist


### How to Review

1. Lookup student rfid:123456
2. Verify it's boaty
3. Lookup student rfid:156
4. Enter 159753 (rufflekins)
5. Enter rfid:156 into search bar, confirm its rufflekins
6. Enter rfid:123
7. Enter 123
8. Create a new student profile
9. Enter info remember rfid!
10. Enter rfid into omnibar and confirm new student.
### Links

- [TFF-205](https://msoese.atlassian.net/browse/TFF-205)
- TheFourFifths/consus/pull/108
